regression-test/suites/segcompaction_p2/test_segcompaction_agg_keys.groovy

```diff
@@ -111,6 +111,51 @@ suite("test_segcompaction_agg_keys") {
 
         String[][] tablets = sql """ show tablets from ${tableName}; """
 
+        def count_src = sql " select count() from ${tableName}; "
+        assertTrue(count_src[0][0] > 0)
+        logger.info("got rows: ${count_src[0][0]}")
+
+        // test partial update
+        def tableName2 = "segcompaction_agg_keys_regression_test_big_table"
```

Comment:
add a data correctness check, e.g. select checksum for every column

Reply:
> add a data correctness check, e.g. select checksum for every column

doris not support select checksum now, so i use md5 function to do checksum, take a look please @zhannngchen